docs/azure-sql-db.md

```diff
@@ -0,0 +1,163 @@
+  ```
+
+  If you can not find the service name, please use the following command to make the plans public.
+
+  ```
+  cf enable-service-access azuresqldb
+  ```
+
+2. Create a service instance
+
+  Configuration parameters are supported with the provision request. These parameters are passed in a valid JSON object containing configuration parameters, provided either in-line or in a file.
+
+  ```
+  cf create-service azuresqldb $service_plan $service_instance_name -c $path_to_parameters
+  ```
+
+  Supported configuration parameters:
+
+  ```
+{
+    "resourceGroup": "<resource-group>",
+    "sqlServerName": "<sql-server-name>",
+    "sqlServerCreateIfNotExist": true | false,
+    "sqlServerParameters": {
+        "location": "<azure-region-name>",
+        "properties": {
+            "administratorLogin": "<sql-server-admin-name>",
+            "administratorLoginPassword": "<sql-server-admin-password>"
+        }
+    },
+    "sqldbName": "<sql-database-name>",
+    "sqldbParameters": {
+        "location": "<azure-region-name>",
+        "properties": {
+            "collation": "SQL_Latin1_General_CP1_CI_AS | <or-other-valid-sqldb-collation>"
+        }
+    }
+}
+  ```
+
+  For example:
+
+  ```
+  cf create-service azuresqldb basic mysqldb -c /tmp/config.json
+  ```
+
+  ```
+  {
+    "resourceGroup": "sqldbResourceGroup",
+    "sqlServerName": "sqlServerA",
+    "sqlServerCreateIfNotExist": true,
+    "sqlServerParameters": {
+        "location": "westus",
+        "properties": {
+            "administratorLogin": "ulrich",
+            "administratorLoginPassword": "u1r8chP@ss"
+        }
+    },
+    "sqldbName": "sqlDbA",
+    "sqldbParameters": {
+        "location": "westus",
+        "properties": {
+            "collation": "SQL_Latin1_General_CP1_CI_AS"
+        }
+    }
+}
+  ```
+
+NOTE: For a list of collation values valid for use with Azure SQL Database, use this query:
+
+SELECT name, description
+FROM fn_helpCollations()
+
+3. Check the operation status of creating the service instance
+
+  The creating operation is asynchronous. You can get the operation status after the creating operation.
+
+  ```
+  cf service $service_instance_name
+  ```
+
+  For example:
+
+  ```
+  cf service mysqldb
+  ```
+
+[More information](http://docs.cloudfoundry.org/devguide/services/managing-services.html#create).
+
+## Using the services in your application
+
+### Binding
+
+  ```
+  cf bind-service $app_name $service_instance_name
+  ```
+
+  For example:
+
+  ```
+  cf bind-service demoapp mysqldb
+  ```
+
+### Format of Credentials
+
+  Verify that the credentials are set as environment variables
+
+  ```
+  cf env $app_name
+  ```
+
+  The credentials have the following format:
+  
+  ```
+  "credentials": {
+    "administratorLogin": "ulrich",
+    "administratorLoginPassword": "u1r8chP@ss",
+    "sqlServerName": "sqlServerA",
+    "sqldbName": "sqlDbA"
+  }
+
+  ```
```

Comment:
Yes. From the devbox.
One option is to allow any IP using `0.0.0.0-255.255.255.255` which is not secure. The ideal way is to allow users to config a rule in the parameter file.